src/core_codemods/lxml_safe_parser_defaults.py

```diff
@@ -0,0 +1,42 @@
+from codemodder.codemods.base_codemod import ReviewGuidance
+from codemodder.codemods.api import SemgrepCodemod
+
+
+class LxmlSafeParserDefaults(SemgrepCodemod):
+    NAME = "safe-lxml-parser-defaults"
+    REVIEW_GUIDANCE = ReviewGuidance.MERGE_AFTER_CURSORY_REVIEW
+    SUMMARY = "Enable all security checks in `lxml.etree.XMLParser` call."
+    DESCRIPTION = "...........TODO"
```

Comment:
Right now this actually gets used as the `CHANGE_DESCRIPTION` for the simplified API (we should fix that). But we do need to make sure it's present.

Reply:
woops total oversight here that's why the TODO. How verbose can I be, should I be?  Is this used as the PR description when pixeebot makes the PR?